### ISSUE TITLE
RSelenium Firefox Fails due to wdman

### ISSUE THREAD

**Issue by nbarsch** [ropensci/wdman#31]
### Operating System
Ubuntu 22.04
### Please describe Expected behaviour
I am getting the error: Error in wdman::selenium(port = port, verbose = verbose, version = version,  : 
  Selenium server  couldn't be started

### Steps to reproduce the behaviour
library(wdman)
rd <- selenium()

**Comment by mfpole** [comment]
I suddenly had an issue too for Firefox (on MacOS), I deleted the "LICENSE.chromedriver" file in the lastest chromedriver file and it worked. 

This is the output of selenium(retcommand = T) before my change, it looks like the license file is causing confusion 

```
"/usr/bin/java c(\"-Dwebdriver.chrome.driver='/Users/griffitj/Library/Application Support/binman_chromedriver/mac64/109.0.5414.25/chromedriver'\",
\"-Dwebdriver.chrome.driver='/Users/griffitj/Library/Application Support/binman_chromedriver/mac64/109.0.5414.25/LICENSE.chromedriver'\") 
-Dwebdriver.gecko.driver='/Users/griffitj/Library/Application Support/binman_geckodriver/macos/0.32.0/geckodriver' -Dphantomjs.binary.path='/Users/griffitj/Library/Application Support/binman_phantomjs/macosx/2.1.1/phantomjs-2.1.1-macosx/bin/phantomjs' -jar '/Users/griffitj/Library/Application Support/binman_seleniumserver/generic/4.0.0-alpha-2/selenium-server-standalone-4.0.0-alpha-2.jar' -port 4567"
```

**Comment by mfpole** [comment]
this is the output after I deleted 

`"/usr/bin/java -Dwebdriver.chrome.driver='/Users/griffitj/Library/Application Support/binman_chromedriver/mac64/109.0.5414.25/chromedriver' -Dwebdriver.gecko.driver='/Users/griffitj/Library/Application Support/binman_geckodriver/macos/0.32.0/geckodriver' -Dphantomjs.binary.path='/Users/griffitj/Library/Application Support/binman_phantomjs/macosx/2.1.1/phantomjs-2.1.1-macosx/bin/phantomjs' -jar '/Users/griffitj/Library/Application Support/binman_seleniumserver/generic/4.0.0-alpha-2/selenium-server-standalone-4.0.0-alpha-2.jar' -port 4567"`

**Comment by nbarsch** [comment]
THIS WORKS! How did you determine the chromedriver license file was the issue?

**Comment by mfpole** [comment]
I can’t find the thread but another user was able to pinpoint a issue arising from multiple files appearing (something about Chrome adding support for M1) where there should only be one so I went to check the driver folders. 

This check step for the driver seems to assume that there is only one file in each path

The output from retcommand that I pasted is showing an R vector for Chrome file path “c(path1,path2)” but the cmd line can’t understand “c()” so deleting the new file gets the new command to be written with the chrome driver path not as a vector

**Comment by mfpole** [comment]
I had this problem reappear today due to a new version of Chrome being downloaded 

Of course, deleting the new LICENSE.chromedriver file will fix the issue again.  However, a more robust fix for now would be to edit your code to set chromever = NULL to prevent wdman from looking for the chrome driver at all. Also, setting check = F will stop wdman from finding new versions and recreating the issue. 

```
library(netstat)
library(RSelenium)
port <- netstat::free_port()
rD <<- rsDriver(browser = "firefox"
                                  ,geckover=<your preference>
                                  ,chromever=NULL
                                  ,check = F  
                                  ,port = port
                                  ,verbose = T
)
```

Additionally, I submitted a PR to this package that would fix the issue by updating the regex. I have no idea how to submit a package update so if someone like @juyeongkim (or anyone else who knows how to maintain a package) could merge and push that would be neat! 

#32

**Comment by charliebone** [comment]
Thanks for submitting the PR. I encountered this issue yesterday and of course I only found this thread after spending the time to troubleshoot and fix the issue. While deleting the LICENSE files is a fairly easy fix I'd definitely appreciate it if we could get that PR merged in and a new package released so things can go back to being fully automated...

**Comment by AlexanderHolmes0** [comment]
I have to say after HOURS of searching the web for an answer this works! Thank you for submitting this.

**Comment by jonathanvoelkle** [comment]
closed in #32
